src/arbitrum/compounders/gmx/GlpCompounder.sol

```diff
@@ -43,17 +42,15 @@ contract GlpCompounder is TokenCompounderBase {
     /// @notice The address of the contract that needs an approval before minting GLP.
     address public glpManager;
 
-    /// @notice The address of sGLP token
+    /// @notice The address of sGLP token.
     address public constant sGLP = 0x5402B5F40310bDED796c7D0F3FF6683f5C0cFfdf;
-    /// @notice The address of WETH token
+    /// @notice The address of WETH token.
     address public constant WETH = 0x82aF49447D8a07e3bd95BD0d56f35241523fBab1;
-    /// @notice The address representing ETH
-    address internal constant ETH = 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE;
```

Comment:
The changes here look like they were introduced in another pull request and now removed again. Is this some unfortunate Git magic or how does this make sense within the scope of the PR title?

Reply:
Nah I fucked it up. Fixed in https://github.com/FortressFinance/fortress-contracts/commit/322fb060449c5c1729554b1e6a5d1c51da32ecd6